packages/timezone/test/timezonePickerTests.tsx

```diff
@@ -236,24 +231,15 @@ describe("<TimezonePicker>", () => {
         assert.lengthOf(span, 1, "expected custom target with class '.foo'");
     });
 
-    function findSelect(timezonePicker: TimezonePickerShallowWrapper) {
+    function findSelect(timezonePicker: TimezonePickerWrapper) {
         return timezonePicker.find(Select.ofType<TimezoneItem>());
     }
 
-    function findQueryList(timezonePicker: TimezonePickerShallowWrapper) {
-        return findSelect(timezonePicker).shallow().find(QueryList.ofType<TimezoneItem>());
-    }
-
-    function findPopover(timezonePicker: TimezonePickerShallowWrapper) {
-        /* eslint-disable-next-line deprecation/deprecation */
-        return findQueryList(timezonePicker).shallow().find(Popover);
-    }
-
-    function findInputGroup(timezonePicker: TimezonePickerShallowWrapper) {
-        return findQueryList(timezonePicker).shallow().find(InputGroup);
+    function findPopover(timezonePicker: TimezonePickerWrapper) {
+        return timezonePicker.find(Popover2);
     }
 
-    function clickFirstMenuItem(timezonePicker: TimezonePickerShallowWrapper): void {
-        findQueryList(timezonePicker).shallow().find(MenuItem).first().simulate("click");
```

Comment:
note: this was a bit of a tricky fix; for some reason this click wasn't working on the MenuItem, I found it better to select for an HTML element with a class name and target that directly rather than selecting a custom React component. I think this advice generalizes to all kinds of unit tests where we are not inspecting the component props/state (which... is another conversation, we probably shouldn't be relying on those kinds of implementation details in tests)